src/main/java/com/powsybl/openloadflow/OpenLoadFlowProvider.java

```diff
@@ -88,12 +88,6 @@ private static AcLoadFlowObserver getObserver(OpenLoadFlowParameters parametersE
                                prefix + "status", result.getNewtonRaphsonStatus().name());
     }
 
-    private static ImmutableMap<String, String> createMetrics(DcLoadFlowResult result) {
```

Comment:
If you remove that, it means that we have no metrics at the end of the computation using itools or pypowsybl. The number of iterations is not needed as it means nothing but the status is usefull.

Reply:
Now there is a way to fill status and iteration on component result these metrics are useless ( and this is true also for ac)

Reply:
Metrics might still be useful fior ourput infos not yet normmilzed in api output